src/app/store/actions/employee.actions.ts

```diff
@@ -43,9 +44,11 @@ export const ActivateEmployeeSuccess = createAction(EmployeesActions.ActivateEmp
 
 export const ReceivedFailure = createAction(EmployeesActions.ReceivedFailure, props<{ error: string | null }>());
 
-export const GetEmployeesPermissions = createAction(EmployeesActions.GetEmployeesPermissions, props<{ email: string; reset: boolean }>());
+export const GetEmployeesPermissions = createAction(EmployeesActions.GetEmployeesPermissions, props<{ email: string }>());
 
 export const GetEmployeesPermissionsSuccess = createAction(
   EmployeesActions.GetEmployeesPermissionsSuccess,
-  props<{ positionsAuthorities: EmployeePositionsAuthorities; reset: boolean }>()
+  props<{ positionsAuthorities: EmployeePositionsAuthorities }>()
 );
+
+export const ResetEmployeePermissions = createAction(EmployeesActions.ResetEmployeePermissions, props<{ reset: boolean }>());
```

Comment:
Is it necessary props here? It is not used in reducer and it does not have sense when it is false